project/src/components/add-review-form/add-review-form.tsx

```diff
@@ -1,42 +1,54 @@
+import { useState } from 'react';
+
 type RatingInputProps = {
   name: string;
   index: number;
+  onChecked: (id: string) => void;
```

Comment:
```suggestion
  onRatingChange: (id: string) => void;
```
так снаружи в родителе будет понятно что делает коллбек